app/src/main/java/de/danoeh/antennapod/fragment/preferences/StoragePreferencesFragment.java

```diff
@@ -210,6 +262,12 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 ab.show();
             }
         }
+
+        if (resultCode == Activity.RESULT_OK &&
+                requestCode == CHOOSE_OPML_EXPORT_PATH) {
```

Comment:
The line does not need to be wrapped. It's short enough to fit in a single line.

Reply:
Fixed!